pallets/settlement/src/benchmarking.rs

```diff
@@ -1006,168 +463,119 @@ benchmarks! {
         assert!(Module::<T>::receipts_used(&signer.account(), 0), "Settlement: change_receipt_validity didn't work");
     }
 
-    reschedule_instruction {
-        let l = T::MaxNumberOfFungibleAssets::get() as u32;
+    //reschedule_instruction {}: _()
+    //verify {}
```

Comment:
Don't we still need a benchmark for `reschedule_instruction`?

Reply:
I added it back. This should take the number of legs as a parameter, but I'll address this in the reschedule issue